examples/update-readme.rs

```diff
@@ -1,10 +1,10 @@
-// Update the "comrak --help" text in Comrak's own README.
+// Update the "comrak --help" text in 's own README.
```

Comment:
A reminder to me to fix this up pre-merge and do another once-over.

```suggestion
// Update the "comrak --help" text in Comrak's own README.
```